### PR TITLE
ADD homopolish_0.0.1

### PR DESCRIPTION
**Pull request by chengjun109**
Homopolish is a Nanopore-specific polisher able to generate a high-quality genome (>Q40) for virus, bacteria, and fungus. The strength of Homopolish lies in correcting Nanopore systematic errors by training a polishing model using homologous sequences retrieved from closely-related genomes. In conjunction with Racon or Medaka, the genomes can reach Q40-90 (>99.99%) accuracy on R9.4 flowcells (Guppy 3.x)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by chengjun109** [comment]
@BiocondaBot please add label

File: recipes/homopolish/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  noarch: python
+  script: python -m pip install . -vv
+requirements:
+    host:
+         - pip
+         - numpy=1.18.5
+    run:
+         - mash=2.1.1
+         - scikit-learn=0.21.3
+         - minimap2=2.17
```
Comment (dpryan79) [inline recipes/homopolish/meta.yaml:23]: Does it really require these exact versions or are later versions also acceptable? I assume all of the `=` can be replaced with `>=`.

Comment (chengjun109) [reply recipes/homopolish/meta.yaml:24]: Yes, We are worried  about the conflict of the environment.
Additionally, some parameters are not applicable in new version.

**Review by dpryan79 — APPROVED** [review]

**Comment by dpryan79** [comment]
@bioconda-bot please merge

**Pull request merged**